src/components/topics-list.component.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { TouchableOpacity, FlatList, Text, View } from 'react-native';
+import { colors, fonts } from 'config';
+
+type Props = {
+  containerStyle: Object,
+  title: string,
+  topics: Array,
+};
+
+const styles = {
+  container: {
+    marginTop: 30,
+  },
+  sectionTitle: {
+    color: colors.black,
+    ...fonts.fontPrimaryBold,
+    marginBottom: 10,
+    paddingLeft: 15,
+  },
+  topic: {
+    backgroundColor: colors.primaryDark,
```

Comment:
I feel like the black background is a bit too strong in my opinion.
This is a secondary information and shouldn't catch the eye that much.

Github uses a light blue
<img width="454" alt="screen shot 2017-11-10 at 9 14 17 am" src="https://user-images.githubusercontent.com/304450/32648752-94224b0c-c5f7-11e7-8884-d21ba53a6ce4.png">
maybe we could use something similar, or a light gray.

But that's really just a matter of taste 😊